packages/server/test/support/fixtures/projects/e2e/cypress/integration/fast_visit_spec.coffee

```diff
@@ -0,0 +1,22 @@
+beforeEach ->
+  cy.on "log:added", (attrs, log) =>
+    if attrs.name is "visit"
+      @lastLog = log
+
+  return null
+
+it "normally finishes in less than 1000ms on localhost with connection: close", ->
+  Cypress._.times 100, ->
+    cy.visit('/close')
+    .then ->
+      expect(@lastLog.get("totalTime")).to.be.lte(1000)
```

Comment:
`1000` seems way too high. shouldn't it be more like <100ms ?

Reply:
The longest request (with the exception of the first request, which took 66ms) finished in less than 25ms on my system. I'll make it 150ms just to prevent any flake from high CPU load in CI or whatever.